### PR TITLE
Vercel CI

### PR DESCRIPTION
**Pull request by peach3es**
using netlify is proving to be a pain in the butt so i'm giving a chance to vercel

### CODE REVIEW

**Comment by peach3es** [comment]
Summary on what I'm trying to do:

I fixed the merge conflicts. Essentially I Remade a Remix project using the latest version cuz apparently they updated it so it kinda changed everything. I put all the files that I created for the CGD website into the new remix folder and tested it locally with "npm run dev" and everything works accordingly. 
As I mentioned above, the integration with Netlify and remix isn't at its best right now since remix JUST updated their version changing everything so all the documentations on using Remix and Netlify are incorrect. I spent the whole day trying to figure it out but couldn't. 
I'm going to give it a try with Vercel

**Comment by maxxfreund** [comment]
Oh okay I'm glad it's working locally,, does the new remix folder work sort of as a new root?
Like is the whole project now in that folder?

File: LICENSE
Comment (maxxfreund) [inline LICENSE:1]: do we still need this licence?

**Comment by peach3es** [comment]
it's the same structure as the last one just the Remix team changed some functionalities around. so yeah file organization wise and root folder remains the same

**Review by maxxfreund — CHANGES_REQUESTED** [review]
dont delete da license pls <3

**Review by peach3es — COMMENTED** [review]
Cool cool cool -Abed

**Review by maxxfreund — APPROVED** [review]
damn thats crazy i saw you do it live

**Pull request merged**